src/settings/lib/color-palette.scss

```diff
@@ -9,8 +9,11 @@ $tbds-warn-yellow: #fbf1a9 !default;
 $tbds-failure-red: #ffcfcf !default;
 $tbds-danger-red: $tbds-failure-red !default;
 
+$tbds-color-white: #fff !default;
+$tbds-color-black: #000 !default;
+
 $tbds-color-gray-900: #111 !default;
-$tbds-color-gray-800: #333 !default;
+$tbds-color-gray-800: $tbds-color-brand-gray !default;
```

Comment:
This is a little weird to me, because I would assume that the variable references `#333` based on its surroundings. What's the purpose of using the brand gray for this value?

Reply:
I like the idea of using a grayscale that is based on our color space. I am going to play with that idea.

Reply:
![Screen Shot 2019-08-16 at 12 00 01 PM](https://user-images.githubusercontent.com/570184/63181049-66a22e00-c01d-11e9-9dcb-ac7799bfd67e.png)
 I updated the colors to be based on our Brand Gray and here's a preview of the SASS variables in my editor.